extensions/tmtv/src/getHangingProtocolModule.js

```diff
@@ -16,37 +16,87 @@ const ptCT = {
   imageLoadStrategy: 'interleaveTopToBottom', // "default" , "interleaveTopToBottom",  "interleaveCenter"
   protocolMatchingRules: [
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'ModalitiesInStudy',
       constraint: {
         contains: ['CT', 'PT'],
       },
-      required: false,
     },
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'StudyDescription',
       constraint: {
-        contains: {
-          value: 'PETCT',
-        },
+        contains: 'PETCT',
       },
-      required: false,
     },
     {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
       attribute: 'StudyDescription',
       constraint: {
-        contains: {
-          value: 'PET/CT',
-        },
+        contains: 'PET/CT',
       },
```

Comment:
why we are removing the `id` and `weigth` and `required`? I assume weigth and required have defaults of `1` and `false`, but what about `id`?

Reply:
Why do you need an id?  It isn't used anywhere.  Every item is referred to positionally.
weight defaults to 1 and required to false.